File: Final 2018-3/srcgen/Telefono.cpp
class Telefono{

  Telefono(this){
  }
  Telefono((this,indicador,numero):
      this.indicador=indicador
      this.numero=numero

  void getIndicador(){
    return this.indicador
	}
  void getNumero(){
    return this.numero
	}
  String setIndicador(this,indicador){
    this.indicador = indicador
  }

  String setNumero(this,numero){
    this.numero = numero
  }

};

File: Final 2018-3/srcgen/Telefono.rb
	
class Telefono

	def initialize(this,indicador,numero)
		@indicador = indicador
		@numero = numero
	end
	
	def indicador
		return @indicador
	end

	def numero
		return @numero
	end

    def indicador=(indicador)
		@indicador = indicador
    end
	
    def numero=(numero)
		@numero = numero
    end
	
end